conda-envs/environment-test-py37.yml

```diff
@@ -0,0 +1,31 @@
+name: pymc3-test-py37
+channels:
+- conda-forge
+- defaults
+dependencies:
+- aesara>=2.1.0
+- arviz>=0.11.2
+- cachetools>=4.2.1
+- cloudpickle
+- fastprogress>=0.2.0
+- h5py>=2.7
+- ipython>=7.16
+- libblas=*=*mkl
+- mkl-service
+- myst-nb
+- numpy=1.15
+- numpydoc>=0.9
+- pandas=0.24
```

Comment:
Do we really need to pin pandas to such an old version?

Below it doesn't even pin a version. Is this because of some Python 3.7 oddity?

Reply:
IIRC there was one env which would have the deps pinned to the earliest allowed versions

Reply:
and I believe that pinning minimum versions helps and speeds up the conda dependency resolver, right?

Reply:
I don't know